eksupgrade/starter.py

```diff
@@ -30,11 +29,10 @@
     find_node,
     is_cluster_auto_scaler_present,
     unschedule_old_nodes,
-    update_addons,
 )
 from .src.latest_ami import get_latest_ami
 from .src.preflight_module import pre_flight_checks
-from .src.self_managed import filter_node_groups, get_asg_node_groups, get_node_groups, update_nodegroup
+from .src.self_managed import get_asg_node_groups, update_nodegroup
```

Comment:
- Not using `get_asg_node_groups` & `get_asgs`. Safe to remove ?

Reply:
Removed!  A long with the now unused logic :D